Content.Client/UserInterface/Systems/Chat/ChatUIController.cs

```diff
@@ -546,7 +546,7 @@ private void UpdateChannelPermissions()
         }
 
         // only admins can see / filter asay
-        if (_admin.HasFlag(AdminFlags.Admin))
+        if (_admin.HasFlag(AdminFlags.Adminhelp))
         {
             FilterableChannels |= ChatChannel.Admin;
             FilterableChannels |= ChatChannel.AdminAlert;
```

Comment:
Codermins should be able to read admin chat but do not have ahelp perms.

Reply:
Should be fine now, at least it doesn't break anything.